esignet-default.properties

```diff
@@ -243,6 +243,7 @@ mosip.esignet.cache.expire-in-seconds={'clientdetails' : 86400, \
 
 mosip.esignet.domain.url=https://${mosip.esignet.host}
 mosip.esignet.discovery.issuer-id=${mosip.esignet.domain.url}${server.servlet.path}
+mosip.esignet.file.url=https://${mosip.esignet.host}
```

Comment:
why are we creating this variable ? why not use the mosip.esignet.domain.url ??

Reply:
changed